pkg/flowaggregator/exporter/s3.go

```diff
@@ -80,7 +80,7 @@ func (e *S3Exporter) UpdateOptions(opt *options.Options) {
 	}
 	if config.BucketName != e.s3UploadProcess.GetBucketName() ||
 		config.BucketPrefix != e.s3UploadProcess.GetBucketPrefix() ||
-		config.Region == e.s3UploadProcess.GetRegion() {
+		config.Region != e.s3UploadProcess.GetRegion() {
```

Comment:
I know it's a one line fix, but let's have it in a separate PR.
Thanks for catching this BTW.

Reply:
done